offline_signing/transfer.sh

```diff
@@ -135,9 +134,26 @@ roll() {
         --version="${version}"
 }
 
+roll() {
+    local channel="$1"
+    local seconds="$2"
+    local board="$3"
+
+    updateservicectl \
+        --server="https://public.update.core-os.net" \
+        --user="${ROLLER_USERNAME}" \
+        --key="${ROLLER_API_KEY}" \
+        rollout create linear \
+        --app-id="${APPID[${board}]}" \
+        --group-id="${channel}" \
+        --duration="${seconds}" \
+        --frame-size="60"
+}
+
 usage() {
     echo "Usage: $0 {download|upload} <ARTIFACT-DIR> [{-a|-b|-s} <VERSION>]..." >&2
-    echo "Usage: $0 roll [{-a|-b|-s} <VERSION>]..." >&2
+    echo "Usage: $0 ready [{-a|-b|-s} <VERSION>]..." >&2
+    echo "Usage: $0 roll [{-a|-b|-s} <SECONDS-TO-100-PERCENT>]..." >&2
```

Comment:
Please don't make me type `86400`.  :cry:

Reply:
would you rather have it as hours? days? I'm open for whatever here.

Reply:
If it comes to that, we can use the raw updateservicectl commands too.

Reply:
or just modify the script sometime later. I'll set it to hours.

@arithx I agree in general that allowing arbitrary duration units would be ideal, but that's not how I ended up writing it for `updateservicectl` and I have no clue how to do that in bash (and I'm not sure I want to know)